api/src/main/java/org/openmrs/module/htmlformentryui/tag/UiIncludeTagHandler.java

```diff
@@ -99,28 +99,28 @@ public void doEndTag(FormEntrySession formEntrySession, PrintWriter out, Node pa
     protected void doIncludeFragment(Node node, UiUtils uiUtils, FormEntrySession formEntrySession, PrintWriter out, String provider) {
```

Comment:
Rename it to simply `includeFragment` and separate the args as input first, output last:
```java
protected void includeFragment(Node node, UiUtils uiUtils, FormEntrySession formEntrySession, String provider, PrintWriter out) {
```